server/odc-service/src/main/java/com/oceanbase/odc/service/flow/util/DescriptionGenerator.java

```diff
@@ -33,21 +39,30 @@
 public class DescriptionGenerator {
 
     public static void generateDescription(CreateFlowInstanceReq req) {
+        Locale locale = LocaleContextHolder.getLocale();
         if (StringUtils.isEmpty(req.getDescription())) {
-            String descFormat = Symbols.LEFT_BRACKET.i18nKey() + "%s" + Symbols.RIGHT_BRACKET.i18nKey() + "%s.%s";
+            // descriptions is recommended for localization.Facilitate fuzzy query
+            String descFormat = Symbols.LEFT_BRACKET.getLocalizedMessage() + "%s"
+                    + Symbols.RIGHT_BRACKET.getLocalizedMessage() + "%s.%s";
             if (req.getTaskType() == TaskType.MULTIPLE_ASYNC) {
                 MultipleDatabaseChangeParameters parameters = (MultipleDatabaseChangeParameters) req.getParameters();
                 List<DatabaseChangeDatabase> databases = parameters.getDatabases();
                 String description = databases.stream()
-                        .map(db -> String.format(descFormat, db.getEnvironment().getName(),
+                        .map(db -> String.format(descFormat, localEnvName(db.getEnvironment().getName(), locale),
                                 db.getDataSource().getName(), db.getName()))
-                        .collect(Collectors.joining(","));
+                        .collect(Collectors.joining(Symbols.COMMA.getLocalizedMessage()));
                 req.setDescription(description);
             } else {
                 req.setDescription(String.format(descFormat,
-                        req.getEnvironmentName(), req.getConnectionName(), req.getDatabaseName()));
+                        localEnvName(req.getEnvironmentName(), locale), req.getConnectionName(),
+                        req.getDatabaseName()));
             }
         }
     }
 
+    private static String localEnvName(@NotNull String envName, @NotNull Locale locale) {
+        String envKey = envName.substring(2, envName.length() - 1);
```

Comment:
i did not understand why substring operate required here.
and, may extract to EnvironmentName for reuse in MultipleDatabaseChangeTask

Reply:
1. The substring operation is to remove the "${" at the beginning of the string and the "} "at the end of the string to obtain the key.
2. the method will be reused in MultipleDatabaseChangeTask